test/openwpm_jstest.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from ..automation.utilities import db_utils
+from .openwpmtest import OpenWPMTest
+
+
+class OpenWPMJSTest(OpenWPMTest):
```

Comment:
The get_config is a specific subclass unique to the js tests - I was just drying out that repeated block that was all over the place.

I think it's helpful to have it separate because my not secret longer term goal is to move this over to a pytest paradigm which doesn't have classes and uses fixtures and it helps to know what goes with what if we're going to make that change.